CppcheckTargets.cmake

```diff
@@ -125,6 +117,11 @@ function(add_cppcheck _name)
   if(NOT TARGET cppcheck_${PROJECT_NAME})
     add_custom_target(cppcheck_${PROJECT_NAME})
   endif()
+  if(NOT TARGET ${PROJECT_NAME}_tests)
+    add_custom_target(${PROJECT_NAME}_tests)
+  endif()
+
   add_dependencies(cppcheck_${PROJECT_NAME} cppcheck_run_${_name})
+  add_dependencies(${PROJECT_NAME}_tests cppcheck_run_${_name})
```

Comment:
Is there a good reason for having cppcheck_run_${_name} part of two meta-targets..?